tools/docker/builder/openwrt/build.sh

```diff
@@ -117,6 +117,12 @@ main() {
             TARGET_PROFILE=DEVICE_NETGEAR_RAX40
             PRPLMESH_VARIANT="-dwpal"
             ;;
+        axepoint)
+            TARGET_SYSTEM=intel_mips
+            SUBTARGET=xrx500
+            TARGET_PROFILE=DEVICE_AX6000_2000_ETH_11AXUCI
```

Comment:
(nitpick) since it's called ax6000 here, wouldn't it be better to call the device ax6000?